### PR TITLE
Migrate changelog to github-changelog-generator

### PR DESCRIPTION
**Pull request by tobiashuste**
For reference, changelog created via this command:

```console
podman run -it --rm -v "$(pwd)":/usr/local/src/your-app:Z,U githubchangeloggenerator/github-changelog-generator -t <token>
```

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+<!--
+SPDX-FileCopyrightText: 2021 Helmholtz Centre for Environmental Research (UFZ)
+SPDX-FileCopyrightText: 2021 Helmholtz-Zentrum Dresden-Rossendorf (HZDR)
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+## [0.5.1](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/releases/v0.5.1) - 2022-03-17
+
+[List of commits](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/compare/v0.5.0...v0.5.1)
```
Comment (Normo) [inline HISTORY.md:10]:
```suggestion
[List of commits](https://github.com/hifis-net/ansible-role-gitlab-runner/compare/v0.5.0...v0.5.1)
```

File: HISTORY.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+### Fixed
+
+* Fix install from deb file if version is older than installed
+  ([!55](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/merge_requests/55)
+  by [tobiashuste](https://gitlab.com/tobiashuste)).
+
+## [0.5.0](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/releases/v0.5.0) - 2022-03-16
+
+[List of commits](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/compare/v0.4.0...v0.5.0)
```
Comment (Normo) [inline HISTORY.md:20]:
```suggestion
[List of commits](https://github.com/hifis-net/ansible-role-gitlab-runner/compare/v0.4.0...v0.5.0)
```

File: HISTORY.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+### Added
+
+* Add support for optionally installing gitlab-runner via a `.deb` file
+  ([!53](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/merge_requests/53)
+  by [tobiashuste](https://gitlab.com/tobiashuste)).
+
+## [0.4.0](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/releases/v0.4.0) - 2022-03-03
+
+[List of commits](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/compare/v0.3.0...v0.4.0)
```
Comment (Normo) [inline HISTORY.md:30]:
```suggestion
[List of commits](https://github.com/hifis-net/ansible-role-gitlab-runner/compare/v0.3.0...v0.4.0)
```

File: HISTORY.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+### Changed
+
+* Allow to update the installed GPG keys
+  ([!52](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/merge_requests/52)
+  by [tobiashuste](https://gitlab.com/tobiashuste)).
+
+## [0.3.0](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/releases/v0.3.0) - 2022-01-11
+
+[List of commits](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/compare/v0.2.2...v0.3.0)
```
Comment (Normo) [inline HISTORY.md:40]:
```suggestion
[List of commits](https://github.com/hifis-net/ansible-role-gitlab-runner/compare/v0.2.2...v0.3.0)
```

File: HISTORY.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+
+### Fixed
+
+* Fix debian docker issue in GitLab CI and bump runner version in tests
+  ([!49](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/merge_requests/49)
+  by [tobiashuste](https://gitlab.com/tobiashuste)).
+
+## [0.2.2](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/releases/v0.2.2) - 2021-07-30
+
+[List of commits](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/compare/v0.2.1...v0.2.2)
```
Comment (Normo) [inline HISTORY.md:68]:
```suggestion
[List of commits](https://github.com/hifis-net/ansible-role-gitlab-runner/compare/v0.2.1...v0.2.2)
```

File: HISTORY.md [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+
+### Fixed
+
+* Fix failing binfmt-init.service for multiarch support
+  ([!44](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/merge_requests/44)
+  by [tobiashuste](https://gitlab.com/tobiashuste)).
+
+## [0.2.1](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/releases/v0.2.1) - 2021-07-30
+
+[List of commits](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/compare/v0.2.0...v0.2.1)
```
Comment (Normo) [inline HISTORY.md:78]:
```suggestion
[List of commits](https://github.com/hifis-net/ansible-role-gitlab-runner/compare/v0.2.0...v0.2.1)
```

File: HISTORY.md [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+* Correctly configure MTU and registry mirror for Docker-in-Docker
+  ([!43](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/merge_requests/43)
+  by [tobiashuste](https://gitlab.com/tobiashuste)).
+* Fix bug when `cache_type` is undefined
+  ([!42](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/merge_requests/42)
+  by [tobiashuste](https://gitlab.com/tobiashuste)).
+
+## [0.2.0](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/releases/v0.2.0) - 2021-07-28
+
+[List of commits](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/compare/v0.1.0...v0.2.0)
```
Comment (Normo) [inline HISTORY.md:91]:
```suggestion
[List of commits](https://github.com/hifis-net/ansible-role-gitlab-runner/compare/v0.1.0...v0.2.0)
```

File: HISTORY.md [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+  by [tobiashuste](https://gitlab.com/tobiashuste)).
+
+### Fixed
+* Document the `limit` parameter
+  ([!40](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/merge_requests/40)
+  by [tobiashuste](https://gitlab.com/tobiashuste)).
+
+## [0.1.0](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/releases/v0.1.0) - 2021-07-20
+
+[List of commits](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/compare/359ac4d5e6371452d5488fcf7daa3a43d935ddc1...v0.1.0)
```
Comment (Normo) [inline HISTORY.md:116]:
```suggestion
[List of commits](https://github.com/hifis-net/ansible-role-gitlab-runner/compare/359ac4d5e6371452d5488fcf7daa3a43d935ddc1...v0.1.0)
```

File: HISTORY.md [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+* Skip ignition check tasks if flatcar config is unchanged
+  ([!37](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/merge_requests/37)
+  by [tobiashuste](https://gitlab.com/tobiashuste)).
+
+### Fixed
+* Document the `limit` parameter
+  ([!40](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/merge_requests/40)
+  by [tobiashuste](https://gitlab.com/tobiashuste)).
+
+## [0.1.0](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/releases/v0.1.0) - 2021-07-20
```
Comment (Normo) [inline HISTORY.md:114]:
```suggestion
## [0.1.0](https://github.com/hifis-net/ansible-role-gitlab-runner/releases/v0.1.0) - 2021-07-20
```

File: HISTORY.md [hunk 10]
```diff
@@ -0,0 +1,119 @@
...
+### Fixed
+
+* Correctly configure MTU and registry mirror for Docker-in-Docker
+  ([!43](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/merge_requests/43)
+  by [tobiashuste](https://gitlab.com/tobiashuste)).
+* Fix bug when `cache_type` is undefined
+  ([!42](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/merge_requests/42)
+  by [tobiashuste](https://gitlab.com/tobiashuste)).
+
+## [0.2.0](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/releases/v0.2.0) - 2021-07-28
```
Comment (Normo) [inline HISTORY.md:89]:
```suggestion
## [0.2.0](https://github.com/hifis-net/ansible-role-gitlab-runner/releases/v0.2.0) - 2021-07-28
```

File: HISTORY.md [hunk 11]
```diff
@@ -0,0 +1,119 @@
...
+
+[List of commits](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/compare/v0.2.1...v0.2.2)
+
+### Fixed
+
+* Fix failing binfmt-init.service for multiarch support
+  ([!44](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/merge_requests/44)
+  by [tobiashuste](https://gitlab.com/tobiashuste)).
+
+## [0.2.1](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/releases/v0.2.1) - 2021-07-30
```
Comment (Normo) [inline HISTORY.md:76]:
```suggestion
## [0.2.1](https://github.com/hifis-net/ansible-role-gitlab-runner/releases/v0.2.1) - 2021-07-30
```

File: HISTORY.md [hunk 12]
```diff
@@ -0,0 +1,119 @@
...
+  ([!50](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/merge_requests/50)
+  by [tobiashuste](https://gitlab.com/tobiashuste)).
+
+### Fixed
+
+* Fix debian docker issue in GitLab CI and bump runner version in tests
+  ([!49](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/merge_requests/49)
+  by [tobiashuste](https://gitlab.com/tobiashuste)).
+
+## [0.2.2](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/releases/v0.2.2) - 2021-07-30
```
Comment (Normo) [inline HISTORY.md:66]:
```suggestion
## [0.2.2](https://github.com/hifis-net/ansible-role-gitlab-runner/releases/v0.2.2) - 2021-07-30
```

File: HISTORY.md [hunk 13]
```diff
@@ -0,0 +1,119 @@
...
+
+[List of commits](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/compare/v0.3.0...v0.4.0)
+
+### Changed
+
+* Allow to update the installed GPG keys
+  ([!52](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/merge_requests/52)
+  by [tobiashuste](https://gitlab.com/tobiashuste)).
+
+## [0.3.0](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/releases/v0.3.0) - 2022-01-11
```
Comment (Normo) [inline HISTORY.md:38]:
```suggestion
## [0.3.0](https://github.com/hifis-net/ansible-role-gitlab-runner/releases/v0.3.0) - 2022-01-11
```

File: HISTORY.md [hunk 14]
```diff
@@ -0,0 +1,119 @@
...
+
+[List of commits](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/compare/v0.4.0...v0.5.0)
+
+### Added
+
+* Add support for optionally installing gitlab-runner via a `.deb` file
+  ([!53](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/merge_requests/53)
+  by [tobiashuste](https://gitlab.com/tobiashuste)).
+
+## [0.4.0](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/releases/v0.4.0) - 2022-03-03
```
Comment (Normo) [inline HISTORY.md:28]:
```suggestion
## [0.4.0](https://github.com/hifis-net/ansible-role-gitlab-runner/releases/v0.4.0) - 2022-03-03
```

File: HISTORY.md [hunk 15]
```diff
@@ -0,0 +1,119 @@
...
+
+[List of commits](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/compare/v0.5.0...v0.5.1)
+
+### Fixed
+
+* Fix install from deb file if version is older than installed
+  ([!55](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/merge_requests/55)
+  by [tobiashuste](https://gitlab.com/tobiashuste)).
+
+## [0.5.0](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/releases/v0.5.0) - 2022-03-16
```
Comment (Normo) [inline HISTORY.md:18]:
```suggestion
## [0.5.0](https://github.com/hifis-net/ansible-role-gitlab-runner/releases/v0.5.0) - 2022-03-16
```

File: HISTORY.md [hunk 16]
```diff
@@ -0,0 +1,119 @@
+<!--
+SPDX-FileCopyrightText: 2021 Helmholtz Centre for Environmental Research (UFZ)
+SPDX-FileCopyrightText: 2021 Helmholtz-Zentrum Dresden-Rossendorf (HZDR)
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+## [0.5.1](https://gitlab.com/hifis/ansible/gitlab-ci-openstack/-/releases/v0.5.1) - 2022-03-17
```
Comment (Normo) [inline HISTORY.md:8]:
```suggestion
## [0.5.1](https://github.com/hifis-net/ansible-role-gitlab-runner/releases/v0.5.1) - 2022-03-17
```

**Review by Normo — CHANGES_REQUESTED** [review]
Thanks @tobiashuste. I like the settings file for github-changelog-generator.  I added some suggestions to move links from gitlab.com to github.com in the `HISTORY.md`. This requires to re-generate the `CHANGELOG.md`.

**Comment by tobiashuste** [comment]
Thanks, @Normo, for fixing all those links! This should be fixed now.

**Review by Normo — APPROVED** [review]
LGTM :+1:

**Pull request merged**